tests/update_files.py

```diff
@@ -157,90 +57,25 @@ def main():
     elif run_test_suite != 'y':
         print('Invalid answer. Exiting.')
         sys.exit(1)
+    else:
+        print()
+
+    # iterate over the given tests directory and find all files named
+    # `test_experiment_*.py` and get their suffixes for use with the
+    # FileUpdater object.
+    suffixes = [re.sub(r'test_experiment_', '', p.stem)
```

Comment:
Is it really necessary to have the `_AUTO_UPDATE` global variables in each test module. Can't we just ignore `rsmcompare` here?

Reply:
It goes back to your other comment for the other things: what if we want to add more modules in the future we don't want to update? I am happy to hardcode it if you guys prefer.

Reply:
No, that makes sense. 

Is there somewhere we can document the we have to add `_AUTO_UPDATE` at the top if/when we add new `test_experiment_*.py` files? I'm just worried I'll totally forget about it. :)

Reply:
Yes, it will be in the testing document that I will be writing.

Reply:
Will you also add to the documentation that the test script should be named `test_experiment_*`?

Reply:
Yes it’s in there already.